projects/cashmere-examples/src/lib/progress-indicator-overview/progress-indicator-overview-example.css

```diff
@@ -0,0 +1,40 @@
+/*.progress-example {*/
```

Comment:
Any way we could've left these as .scss, and avoided commenting them out? Left as is probably would work just fine in a stackblitz, which is our intended target anyways.

Reply:
I probably could, but I want to keep the styling agnostic in-case people chose to go with Less or something other than Sass. I can take a look at that in another PR. I really just want to get this thing merged so I don't have to deal with complicated merges.